src/brasil/gov/paginadestaque/traversal/hooks.py

```diff
@@ -68,5 +71,6 @@ def microsite_expiration_enforcer(event):
         if expired and not sm.checkPermission(ModifyPortalContent, object):
             portal = api.portal.get()
             expires_redirect = getattr(object, 'expires_redirect', '').strip()
-            expires_redirect = expires_redirect if expires_redirect else portal.absolute_url()
+            expires_redirect = (expires_redirect if expires_redirect
```

Comment:
This is to avoid errors in some editors that automatically search for flake8 errors. I don't think there's a problem with these simple comments, they shouldn't be a blocker to merge this pull request.